tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
+    # shellcheck disable=SC2046
+    export $(grep -v '^#' .env | xargs -d '\n')
+fi
+
+MAKE='make --debug=v --output-sync'
+
+# build and push rook image
+$MAKE build BUILD_REGISTRY=local
+build_Image="local/ceph-amd64:latest"
+git_hash=$(git rev-parse --short "${GITHUB_SHA}")
+tag_Image=quay.io/ocs-dev/rook-ceph:v${BRANCH_NAME}-$git_hash
+docker tag "$build_Image" "$tag_Image"
+docker push "$tag_Image"
+
+# build and push rook bundle
+$MAKE bundle
+build_bundle_Image=$(docker images | grep bundle | awk '{print $1; exit}')
+build_bundle_Tag=$(docker images | grep bundle | awk '{print $2; exit}')
+tag_bundle_Image=quay.io/ocs-dev/rook-ceph-operator-bundle:vbundle-${BRANCH_NAME}-$git_hash
+docker tag "$build_bundle_Image":"$build_bundle_Tag" "$tag_bundle_Image"
+docker push "$tag_bundle_Image"
```

Comment:
```suggestion
DOCKERCMD=podman BUNDLE_IMAGE=quay.io/ocs-dev/rook-ceph-operator-bundle:${BRANCH_NAME}-$git_hash  make bundle
docker push quay.io/ocs-dev/rook-ceph-operator-bundle:master-9e5499216 
```
please try this may require little tweak but it should work

Reply:
```
[root@localhost rook]# podman BUNDLE_IMAGE=quay.io/ocs-dev/rook-ceph-operator-bundl  make bundle
Error: unrecognized command `podman BUNDLE_IMAGE=quay.io/ocs-dev/rook-ceph-operator-bundl`
```
If we have variable that can be run with `make cmd` that should work

Reply:
@parth-gr  you missed `DOCKERCMD=podman`  in front

Reply:
Updated,I hope ci knows about podman

Reply:
@travisn @subhamkrai 
We can go ahead and merge if changes look good,

Podman is recognised
```
 podman push quay.io/ocs-dev/rook-ceph:vmerge-988182faf
Error: quay.io/ocs-dev/rook-ceph:vmerge-988182faf: image not known
```
There is no error for the podman, it is just the image is build from the docker so not able to list....

Reply:
great